ambry-api/src/test/java/com.github.ambry/store/MockStoreKeyConverterFactory.java

```diff
@@ -34,7 +33,7 @@ public MockStoreKeyConverterFactory(VerifiableProperties verifiableProperties, M
 
   @Override
   public StoreKeyConverter getStoreKeyConverter() {
-    return storeKeyConverter;
+    return new MockStoreKeyConverter();
```

Comment:
the problem is that the setConversionMap() call is associated with the factory. What does that mean when new MockStoreKeyConverter instances are created?

Reply:
It means all MockStoreKeyConverter's map backings change.  I think it was designed this way because of a test with AmbryRequests (where you can pass it a factory but need to change the mapping) so I didn't change it.

Reply:
As discussed changing MockStoreKeyConverter to support changing an instance's map backing